### ISSUE TITLE
Log installation and removal of add-ons

### ISSUE THREAD

**Issue by hvelarde** [collective/collective.fingerpointing#32]
Products.GenericSetup defines an event ([IProfileImportedEvent](https://github.com/zopefoundation/Products.GenericSetup/blob/1.8.2/Products/GenericSetup/interfaces.py#L819-L828)) that can be used to somehow implement this feature.

**Comment by hvelarde** [comment]
@mauritsvanrees what do you think?

**Comment by mauritsvanrees** [comment]
Yes, this should be doable. Well, this would only log installation of a profile, not of an old-style product with `Extensions/install.py`, but that seems fine with me.

The quick installer tool already uses this event to mark a product as installed in the QI when its profile gets installed, plus recording a snapshot, so that can be used as example:
https://github.com/plone/Products.CMFQuickInstallerTool/blob/3.0.13/Products/CMFQuickInstallerTool/events.py#L70

**Comment by hvelarde** [comment]
thanks, that's exactly what I was looking for!
